unlock_indicator.c

```diff
@@ -261,7 +301,7 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     timeinfo = localtime(&rawtime);
 
     if (unlock_indicator &&
-        (unlock_state >= STATE_KEY_PRESSED || auth_state > STATE_AUTH_IDLE || show_clock)) {
```

Comment:
Now that I've played with it - yeah, I totally see why you broke the indicator and time apart. It looks pretty nice.